pkg/controller/servicebindingrequest/nested/merge.go

```diff
@@ -0,0 +1,12 @@
+package nested
+
+import (
+	"github.com/imdario/mergo"
+)
+
+// WithSmartMerge configures mergo to append slices but not overwrite existing
+// values.
+func WithSmartMerge(config *mergo.Config) {
```

Comment:
this is only used in https://github.com/pedjak/service-binding-operator/blob/3fc9e08015771a1efe6a9a5df352555c6a20fdff/pkg/controller/servicebindingrequest/nested/merge_test.go#L68 Please move it then there.

Reply:
Please submit this change as a PR if you'd like to proceed with this.